src/Spago/RunEnv.hs

```diff
@@ -71,19 +72,22 @@ withPackageSetEnv app = do
 withInstallEnv'
   :: (HasEnv env)
   => Maybe Config
+  -> UsePsa
   -> RIO InstallEnv a
   -> RIO env a
-withInstallEnv' maybeConfig app = do
+withInstallEnv' maybeConfig usePsa app = do
   Env{..} <- getEnv
   envConfig@Config{..} <- case maybeConfig of
     Just c -> pure c
     Nothing -> getConfig
   let envPackageSet = packageSet
+  envPursCmd <- getPurs usePsa
```

Comment:
It looks like most/all the places where we call this pass `NoPsa`, so maybe we should just have it here without having to thread it everywhere?

Reply:
This should be better now that `PursCmd` isn't in `InstallEnv`.